content/chef_repo.md

```diff
@@ -137,30 +86,23 @@ and for vim, do something like:
 
 **Ignore top-level Subversion data**
 
-If using Subversion as the version source control application, it is
-important not to upload certain files that Subversion uses to maintain
-the version history of each file. This is because Chef Infra Client will
-never use it while configuring nodes, plus the amount of data in an
-upload that includes top-level Subversion data could be significant.
+If using Subversion as the version source control application, it is important not to upload certain files that Subversion uses to maintain the version history of each file. This is because Chef Infra Client will never use it while configuring nodes, plus the amount of data in an upload that includes top-level Subversion data could be significant.
```

Comment:
No one in 2021 is using subversion. This should go away

Reply:
I'm always happy to curate the content!